src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

```diff
@@ -81,3 +86,23 @@
         </div>
     }
 </div>
+<script type="text/javascript" at="Foot">
+    togglePassword = document.querySelector('#togglePassword');
+    password = document.querySelector('#Password');
+
+    togglePassword.addEventListener('click', function (e) {
+        // toggle the type attribute
+        type = password.getAttribute('type') === 'password' ? 'text' : 'password';
+        password.setAttribute('type', type);
+
+        // toggle the eye slash icon
+        icon = this.getElementsByClassName('icon')[0];
+        if(icon.getAttribute('data-icon')){ // if the icon is rendered as a svg
+            type === 'password' ? icon.setAttribute('data-icon', 'eye') : icon.setAttribute('data-icon', 'eye-slash');
+        }
+        else{ // if the icon is still a <i> element
+            type === 'password' ? icon.classList.remove('fa-eye-slash') : icon.classList.remove('fa-eye');
```

Comment:
It could be done with single condition instead of two

Reply:
I could have chained the conditional operator but I don't do that normally for the sake of simpler readability mainly... not an issue.